GoogleForCreators/web-stories-wp
# Shopping: Infinite Scroll Pagination

## Context



## Summary

Handle pagination in the shopping panel. Reuse dashboard infinite scroll component. 

## Relevant Technical Choices



## To-do



## User-facing changes

https://user-images.githubusercontent.com/237508/173386049-6a9aa486-cc7f-4f4b-b9d5-19533326269d.mov

## Testing Instructions






This PR can be tested by following these steps:

1. Active shopping with shopify. 
2. Go to editor. 
3. Go to product tab. 
4. Scroll to bottom. 
5. See more products load in. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11593
